src/btree/bt_sync_obsolete.c

```diff
@@ -285,10 +298,10 @@ __checkpoint_cleanup_obsolete_cleanup(WT_SESSION_IMPL *session, WT_REF *parent)
     for (slot = 0; slot < pindex->entries; slot++) {
         ref = pindex->index[slot];
 
-        WT_RET(__sync_obsolete_cleanup_one(session, ref));
+        WT_RET(__sync_obsolete_cleanup_one(session, ref, num_cc_pagesp));
     }
 
-    WT_STAT_CONN_DSRC_INCRV(session, checkpoint_cleanup_pages_visited, pindex->entries);
+    WT_STAT_CONN_DSRC_INCRV(session, cc_pages_visited, pindex->entries);
```

Comment:
Does `cc_pages_visited` necessarily have a valid value?

Reply:
Yes. We are incrementing the stat of `cc_pages_visited` with `pindex->entries`